globals/sanitize-functions.php

```diff
@@ -206,3 +206,132 @@ function neve_sanitize_typography_control( $value ) {
 
 	return $value;
 }
+
+/**
+ * Sanitize alignment.
+ *
+ * @param array $input alignment responsive array.
+ *
+ * @return array
+ */
+function sanitize_alignment( $input ) {
```

Comment:
please either use prefixed functions or namespace, sanitize_alignment is too general

Reply:
I just moved it from here https://github.com/Codeinwp/neve/blob/4186d70efed6b96f9f7476d8451fba6a1c28eb71/header-footer-grid/Core/Components/Abstract_Component.php but now that is global it needs a prefix, you're right. Thanks for the feedback!